### PR TITLE
ENV var for the root folder to rsync to

### PR DESCRIPTION
**Pull request by oudeismetis**
Connected to #16

### CODE REVIEW

**Comment by oudeismetis** [comment]
Had trouble testing this locally.

File: api.go [hunk 1]
```diff
@@ -159,7 +162,7 @@ func Build(buildDir, projectName, remotePath string) string {
 			logging.LogToFile("Container built...transfering built files...")
 			GrabBuiltStaticFiles(buildTag, projectName, buildDir)
 			if Config.ENV != "test" {
-				rsyncProject(buildDir+"/public/*", remotePath)
+				rsyncProject(buildDir+"/public/*", Config.DEPLOYROOTPATH+remotePath)
```
Comment (CorneliusIV) [inline api.go]: `remotePath` should probably be `projectPath`

File: README.md [hunk 2]
```diff
@@ -32,10 +32,12 @@ The following environment variables will need to be set:
 	BUILD_LOCATION
 	API_URL
 	ENV
+  DEPLOY_ROOT_FOLDER
 
 1. Set the BUILD_LOCATION environment variable (location where project will build to): `export BUILD_LOCATION=<location>`
 2. Set the ENV environment variable (production or test): `export ENV=test`
 3. Set the API_URL variable to Franklin API `export API_URL=https://example.com`
+4. Set the DEPLOY_ROOT_FOLDER variable to the folder where successfully build projects will be rsync'd to. Nginx in `franklin`will have the same exact setting.
```
Comment (CorneliusIV) [inline README.md]: Can you give an example of the format here?

File: api.go [hunk 3]
```diff
@@ -43,13 +43,16 @@ func init() {
...
 		panic("Missing environment variable BUILD_LOCATION")
-
 	}
 	Config.FRANKLINAPIURL = os.Getenv("API_URL")
 	if Config.FRANKLINAPIURL == "" {
 		logging.LogToFile("Missing environment variable API_URL")
 		panic("Missing environment variable API_URL")
-
+	}
+	Config.DEPLOYROOTPATH = os.Getenv("DEPLOY_ROOT_FOLDER")
```
Comment (CorneliusIV) [inline api.go:53]: There is a config struct above you need to add `DEPLOYROOTPATH` to that as well.

**Comment by oudeismetis** [comment]
Good catches @CorneliusIV 
Everything should be fixed now.

**Comment by CorneliusIV** [comment]
:+1:

**Pull request merged**